runtime/manifest.md

```diff
@@ -3,100 +3,85 @@ An Arcs Manifest declares collections of Schemas, Particles, Recipes and Views.
 
 By convention a manifest is stored in a file with the `.manifest` extension.
 
-## Indentation
-Indentation is significant. It is used to group blocks of manifest items together. The indentation rules require that items at the same level have the same indentation. By convention this is usually in increments of two spaces.
+## Schemas
+
+Schemas define simple data structures that are passed via Views as inputs and
+outputs of Particles.
```

Comment:
Should we introduce the word Entity too? I think it's clearer than Schema.

Reply:
Added.